Dockerfile

```diff
@@ -43,6 +43,8 @@ ARG PYTHONOPTIMIZE=2
 ARG WKHTMLTOPDF_VERSION=0.12.4
 ARG WKHTMLTOPDF_CHECKSUM='049b2cdec9a8254f0ef8ac273afaf54f7e25459a273e27189591edc7d7cf29db'
 ENV OPENERP_SERVER=/opt/odoo/auto/odoo.conf \
+    ADDON_INSTALL=false \
+    WAIT_CONTAINER=false \
```

Comment:
What if you container is called `false`? OK, it's stupid, but since this is not a boolean value, I prefer to keep it as an empty string/filled string.

Also, call it `WAIT_NOHOST`, since it could be any host that can be resolved, and it is supposed to be unavailable for odoo to start.